compio-driver/src/asyncify.rs

```diff
@@ -0,0 +1,47 @@
+use std::thread::JoinHandle;
+
+use crossbeam_channel::{bounded, Receiver, Sender, TrySendError};
+
+type BoxClosure = Box<dyn FnOnce() + Send>;
+
+fn worker(receiver: Receiver<BoxClosure>) -> impl FnOnce() {
+    move || {
+        while let Ok(f) = receiver.recv() {
+            f();
+        }
+    }
+}
```

Comment:
Add a timeout with `recv_timeout`. Stop the thread if no task were to be received after some time (maybe configurable).